src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -583,7 +583,10 @@ public List<RESTZaakBetrokkene> listBetrokkenenVoorZaak(@PathParam("uuid") final
     @GET
     @Path("communicatiekanalen")
     public List<RESTCommunicatiekanaal> listCommunicatiekanalen() {
-        final List<CommunicatieKanaal> communicatieKanalen = vrlClientService.listCommunicatiekanalen();
+        final List<CommunicatieKanaal> communicatieKanalen = vrlClientService.listCommunicatiekanalen()
```

Comment:
ipv streamen, filteren en tolist aanroepen, is er ook een specifieke operator voor wat je hier wilt bereiken: removeIf

final List<CommunicatieKanaal> communicatieKanalen = vrlClientService.listCommunicatiekanalen();
communicatieKanalen.removeIf(kanaal -> kanaal.getNaam().equals("E-formulier"));

Reply:
Opgelost in meest recente commit